usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -202,10 +212,11 @@ def handle(self, *args, **options):
                     piid=row.get('piid'),
                     fain=row.get('fain'),
                     uri=row.get('uri'),
-                    parent_award_id=row.get('parent_award_id'))
+                    parent_award_id=row.get('parent_award_id'),
+                    use_cache=True)
                 award.latest_submission = submission_attributes
-                award.save()
-            except:
+                award_queue[award.manual_hash()] = award
+            except:   # TODO: silently swallowing a bare exception is bad mojo
```

Comment:
Indeed it is! This is an artifact of an earlier time when we needed something...anything...in our database for testing. I would argue that now is the time to remove big try/catch blocks on our loaders, since we're still loading agency submissions on demand. Let's find the problems now and fix them, before we start running stuff overnight.